website/docs/en/guide/migration/cra.mdx

```diff
@@ -311,7 +311,7 @@ export default {
       resolve: {
         mainFields: ['browser', 'module', 'main'],
       },
-      plugins:
+      appendPlugins:
```

Comment:
Rspack does not provide a `resolve.appendPlugins` config..

Reply:
@chenjiahan my bad. i got confused with this 

![Screenshot 2024-09-26 at 3 52 53 PM](https://github.com/user-attachments/assets/6ef89b6c-f67a-46b4-ad05-3ccfa4b05c71)

https://rsbuild.dev/guide/migration/cra#craco-migration

Reply:
Yes it is a bit confusing, we can change `tools.rspack.appendPlugins` to `tools.rspack#appendPlugins` or something else to make it clearer.